internal/config/config.go

```diff
@@ -89,28 +89,17 @@ func (cfg *GeneralConfig) WriteReport(fileName string, content []byte) error {
 // GetDumpFailedTestReportLocation returns destination file for failed tests logs.
 func (cfg *GeneralConfig) GetDumpFailedTestReportLocation(file string) string {
 	if cfg.DumpFailedTests {
-		return cfg.getDumpReportLocation(file, "failed_")
-	}
-	return ""
-}
-
-// GetDumpTestReportLocation returns destination file for test logs regardless of test status.
-func (cfg *GeneralConfig) GetDumpTestReportLocation(file string) string {
-	return cfg.getDumpReportLocation(file, "failed_")
-}
-
-// getDumpReportLocation is a helper function that creates the report directory if needed
-// and returns the destination file path for test logs.
-func (cfg *GeneralConfig) getDumpReportLocation(file string, prefix string) string {
-	if _, err := os.Stat(cfg.ReportsDirAbsPath); os.IsNotExist(err) {
-		log.Printf("Path to report dir %s does not exist, trying to create now", cfg.ReportsDirAbsPath)
-		err := os.MkdirAll(cfg.ReportsDirAbsPath, 0744)
-		if err != nil {
-			log.Fatalf("panic: Failed to create report dir due to %s", err)
+		if _, err := os.Stat(cfg.ReportsDirAbsPath); os.IsNotExist(err) {
```

Comment:
It would be easier to compare (and fix) if you used GitHubs revert function (a button at the bottom of the original PR).